### ISSUE TITLE
Bean Temperature Can't show on screen

### ISSUE THREAD

**Issue by TonSDe** [artisan-roaster-scope/artisan#720]
**Describe the bug**
BT cannot be displayed on the screen. in spite of data from Modbus.

**To Reproduce**
1. New install Artisan 2.4.6 on macOS
2. Go to -> Config -> Machine -> Coffee-Tech -> Silon Touch
3. Setup my MacBook to IP address to 192.168.1.100
4. Click ON
5. BT show "u.u"

**Expected behavior**

**Screenshots**
Screen 1 : 
<img width="1440" alt="Screen Shot 2564-11-02 at 09 24 15" src="https://user-images.githubusercontent.com/1715801/139779701-93f38eb1-f154-4d45-a1cf-65267437d5fb.png">

Screen 2 : Testing Scan Data from Modbus
<img width="1440" alt="Screen Shot 2564-11-02 at 09 13 25" src="https://user-images.githubusercontent.com/1715801/139779827-fbdce11e-7486-48e2-982f-88e6e954d5ab.png">

Screen 3 : BT Can't show data
<img width="1440" alt="Screen Shot 2564-11-02 at 09 25 45" src="https://user-images.githubusercontent.com/1715801/139779977-6f681552-987e-4c41-aba3-55c644064dfd.png">

**Setup (please complete the following information):**
 - Artisan Version : 2.4.6
 - Artisan Build (number in brackets shown in the about box) [e.g. 97e6dec]:  f1b8e57
 - Version [e.g. 22]: 
 - Platform (Mac/Windows/Linux + OS version) [e.g. macOS 10.15.3]:  macOS 12.0.1
 - Connected devices or roasting machine [e.g. Probatone 5]: Silon ZR-7

**Additional context**
[silon-zr7-artisan-settings.aset.zip](https://github.com/artisan-roaster-scope/artisan/files/7457581/silon-zr7-artisan-settings.aset.zip)

**Comment by MAKOMO** [comment]
Thanks for your detailed report! I did load my MODBUS simulator with the values shown in your second screenshot, started Artisan v2.4.6, loaded your Artisan settings (which look absolutely fine to me), pointed it to my simulator and Artisan showed the correctly decoded data.

<img width="606" alt="Screenshot 2021-11-02 at 20 06 06" src="https://user-images.githubusercontent.com/13909011/139929112-0c8bacb2-02e7-4580-bd3a-eddf80a75149.png">

I also checked if you have any custom symbolic formula for BT defined or any input filter limit set, but couldn't find neither in your settings.

Is the BT permanently not shown? Was it never shown? Is this a new machine? Did you check back with CTE?
